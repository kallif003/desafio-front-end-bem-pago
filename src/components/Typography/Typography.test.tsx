import * as React from "react"
import { render } from "@testing-library/react"
import { H1, H2 } from "./"

describe("typography components", () => {
	it("Renders H1 components", () => {
		const { container } = render(<H1>Test</H1>)
		expect(container.firstChild).toMatchSnapshot()
	})
	it("Renders H2 components", () => {
		const { container } = render(<H2>Test</H2>)
		expect(container.firstChild).toMatchSnapshot()
	})
})
