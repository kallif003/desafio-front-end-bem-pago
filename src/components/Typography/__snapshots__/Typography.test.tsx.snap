// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`typography components Renders H1 components 1`] = `
<h1
  class="sc-bdvvtL bwDMfC"
>
  Test
</h1>
`;

exports[`typography components Renders H2 components 1`] = `
<h2
  class="sc-gsDKAQ fusehV"
>
  Test
</h2>
`;
